src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -241,7 +239,7 @@ public boolean contains(String key) {
                 }
 
                 @Override
-                public Iterator<TokenCacheItem> getAll() {
+                public Iterable<TokenCacheItem> getAll() {
```

Comment:
Got it now, Thanks. But I don't think I want to cover it at this moment. That could be the future improvement as what I mentioned above. What do you think?